.github/workflows/end-to-end.yml

```diff
@@ -3,6 +3,7 @@ name: End-to-End tests
 on:
   schedule:
     - cron: '0 4 * * *' # run at 4 AM UTC
+  workflow_dispatch:
```

Comment:
I assume this is intentional to make it easier to test in the future :-)

Reply:
yep, exactly